app/javascript/components/pages/__test__/AboutUs.test.js

```diff
@@ -13,4 +13,20 @@ describe("When AboutUs renders", () => {
 
     expect(aboutUsHeading.text()).toEqual("About Us");
   });
+
+  it("displays three cardtitles", () => {
+    const aboutUs = shallow(<AboutUs/>);
+
+    const aboutUsCardTitle = aboutUs.find("h5");
+
+    expect(aboutUsCardTitle.length).toEqual(3);
+  })
```

Comment:
if the headings here aren't expected to change, the text for them should be asserted on